src/SAWScript/Crucible/JVM/Builtins.hs

```diff
@@ -314,13 +313,17 @@ verifyObligations cc mspec tactic assumes asserts =
                        , goalName = nm
                        , goalLoc  = gloc
                        , goalDesc = msg
-                       , goalProp = goal'
+                       , goalSequent = propToSequent goal'
                        , goalTags = MS.conditionTags md
                        }
-       res <- runProofScript tactic proofgoal (Just ploc) $ Text.unwords
-                 ["JVM verification condition:", Text.pack (show n), Text.pack goalname]
+       res <- runProofScript tactic goal' proofgoal (Just ploc)
+                (Text.unwords
+                 ["JVM verification condition:", Text.pack (show n), Text.pack goalname])
+                False -- do not record in the theorem database
```

Comment:
Basically, we want to avoid registering verification conditions that arise from `llvm_verify`, `jvm_verify`, etc in the theorem database to avoid keeping references to their (potentially quite large) statements. The VCs generated in this way are never exposed to the user directly, so they can't be used in the the proofs of later theorems. Avoiding registering them is an attempt to reduce memory pressure.

Reply:
Aha! Can you specifically cite `llvm_verify`, `jvm_verify`, etc. as examples of things that shouldn't be recorded in the theorem database in the Haddocks for `SAWScript.Proof.finishProof`?